demo/java/src/main/java/io/github/cosinekitty/astronomy/demo/Main.java

```diff
@@ -25,27 +26,14 @@ private static Time parseTime(String args[], int index) {
             Date now = new Date();
```

Comment:
Is it possible to eliminate this use of `Date` also? Like: `Instant.now()`?

Reply:
Or I can just change it to `System.currentTimeMillis()` like you have below. I will do that because it seems more direct.

Reply:
Yes, `System.currentTimeMillis()` will be better I think so also.